### DISCUSSION TITLE
[v1.2.3]What are these devices for? Can I delete them?

### DISCUSSION THREAD

**Discussion by happywzy** [longhorn/longhorn#5629] [General]
Version：v1.2.3
```
[root@master2 ~]# lsblk 
NAME   MAJ:MIN RM  SIZE RO TYPE MOUNTPOINT
sda      8:0    0   40G  0 disk 
└─sda1   8:1    0   40G  0 part /
sdc      8:32   0    2G  0 disk /var/lib/kubelet/pods/5f7239c5-bd3e-43cc-aca9-9ba9179d5247/volumes/kubernetes.io~csi/pvc-b3b7c3e0-cb59-41c3-873f-1c1d4d93e9b4/mount
sdd      8:48   0   50G  0 disk 
sde      8:64   0   30G  0 disk 
sdf      8:80   0    2G  0 disk 
sdg      8:96   0    5G  0 disk 
```
can I delete sdd sde sdf sdg?
when I create a new pv in the kubernetes use longhorn,  system will not create device in the node .
And how to delete these devices?
thank you.

**Comment by derekbit** [comment]
You can check the device number in `/dev/longhorn/<volume name>` and `/dev/sd[x]`. If the device `/dev/sd[x]` has the same device number as the device in /dev/longhorn/<volume name>, it is a longhorn device. 
The lifecycle of the devices are managed by longhorn system.

**Reply by happywzy** [reply]
![1679464140205](https://user-images.githubusercontent.com/26290710/226813702-f77f04a1-ed0d-4a3e-8d13-b5e44ea52afb.png)

- The device `sdf` on the picture `await` time is bigger than others, Is this normal?
- the CPU load of this node in last 5 minutes will suddenly increases every 7 hours, And then it will fall back to normal range，I'm not sure it is the longhorn task process?
![1679464187798](https://user-images.githubusercontent.com/26290710/226813825-728a576a-d712-41d8-9246-11b7b28419a1.png)
